src/friends.cpp

```diff
@@ -62,13 +62,23 @@ EXTERN int luasteam_getFriendPersonaName(lua_State *L) {
     return 1;
 }
 
+// bool SetRichPresence( const char *pchKey, const char *pchValue );
+EXTERN int luasteam_setRichPresence(lua_State *L) {
+    const char *key = luaL_checkstring(L, 1);
+    const char *value = luaL_checkstring(L, 2);
+    bool success = SteamFriends()->SetRichPresence(key, value);
+    lua_pushboolean(L, success);
+    return 1;
+}
+
```

Comment:
Code seems legit.

Reply:
Yeah, I've tested it out already: https://cdn.discordapp.com/attachments/474705430434807819/784860986032521226/2019-12-05_steam_rich_presence_2.gif

I probably should have mentioned that.